### ISSUE TITLE
FP double_parens: tuples inside assert_eq with message

### ISSUE THREAD

**Issue by matthiaskrgr** [rust-lang-nursery/rust-clippy#3206]
`clippy 0.0.212 (125907a 2018-09-17)`

This works fine
````rust
assert_eq!((1, 2), (1, 2));
````
but as soon as we add message to the assert
````rust
fn main() {
    assert_eq!((1, 2), (1, 2), "oh no!");
}
````
clippy falsely reports unnecessary double parens:
````
cargo clippy -- -W clippy::pedantic


warning: Consider removing unnecessary double parentheses
 --> src/main.rs:2:5
  |
2 |     assert_eq!((1, 2), (1, 2), "oh no!");
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: #[warn(clippy::double_parens)] on by default
  = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)
````
